doc/Motr_Terminology_Guide.rst

```diff
@@ -0,0 +1,172 @@
++---------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ATOM                                                    | Attribute on metadata server.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
++---------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Back-End (BE)                                           | Back-end supports local transactions so that FOL could be populated atomically with other tables.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
@max-seagate and @nikitadanilov Could you please provide the new definitions.

Reply:
Back-end (or simply "BE"): local meta-data back-end of motr. This component maintains transactional storage used by motr services to persistently store meta-data. BE used write-ahead-logging (WAL) to implement transactions. Main interfaces provided by BE are: transactions, allocator and b-tree. BE performance and stability are critical for overall motr performance, because all meta-data (internal and external) are stored and accessed via it.